source/katana/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: home
+sidebar: none
+product: katana
+generator: pagination
+pagination:
+    max_per_page: 5
+use:
+    - posts
+bodyClass: page--katana
+---
+<section class="box box--hero">
+    <div class="box-wrapper">
+        <div class="box-text">
+            <img src="{{ site.url }}/img/logo-katana.png" alt="Sabre Katana">
+            <p class="prodname--katana">sabre/<span>katana</span></p>
+            <h1>The modern and lightweight CardDAV and CalDAV server.</h1>
+            <a class="install" href="{{ site.url }}/katana/install">
+                <i class="fa fa-download"></i>
+                Install
+            </a>
+        </div>
+        <div class="box-list">
+        <ul class="sheet-features">
+            <li>
+                <i class="fa fa-refresh fa-fw"></i>
+                Contact, calendar & task syncing via CardDAV/CalDAV.
+            </li>
+            <li>
+                <i class="fa fa-magic fa-fw"></i>
+                Easy installation
+            </li>
+            <li>
+                <i class="fa fa-sitemap fa-fw"></i>
+                Modern architecture
+            </li>
+            <li>
+                <i class="fa fa-users fa-fw"></i>
+                Built-In user management.
+            </li>
+            <li>
+                <a href="{{ site.url }}/katana/install">
+                    <i class="fa fa-file-text fa-fw"></i>
+                    Awesome documentation
+                </a>
+            </li>
+        </ul>
+        </div>
+    </div>
+    <div class="box-shadow"></div>
+</section>
+
+<section class="box box--sandbottom text--center">
+    <h1>Manage your own server</h1>
+    <figure class="image-cutbottom">
+        <img src="{{ site.url }}/img/screen-katana-admin.png" alt="Log in screen for sabre/katana">
+    </figure>
+    <div class="box-shadow"></div>
+</section>
+
+<section class="box box--ice text--center">
+    <h1>Powered by</h1>
+    <a class="logo" href="{{ site.url }}/">
+        <img src="{{ site.url }}/img/logo.png" alt="sabre/dav">
+        <br>
+        sabre<span>/dav</span>
+    </a>
+        <h2>The leading CardDAV, CalDAV and WebDAV framework.</h2>
+        <a href="http://sabre.io" class="btn">sabre.io</a>
+    <div class="panel panel--t panel--trustedby">
+        <p>Proven technology that is trusted by:</p>
```

Comment:
These are the links I'd like to drop. They don't even really apply to the katana bit.